source/views/building-hours/report/editor.tsx

```diff
@@ -31,146 +32,127 @@ type Props = TopLevelViewPropsType & {
 		state: {
 			params: {
 				set: SingleBuildingScheduleType
-				onEditSet: (set: SingleBuildingScheduleType) => any
-				onDeleteSet: () => any
+				onEditSet: (set: SingleBuildingScheduleType) => unknown
+				onDeleteSet: () => unknown
```

Comment:
Should these be anonymous functions that return void rather than unknown?

Reply:
By allowing them to return `unknown`, we allow ourselves to pass anything in without worrying about returning nothing at the end - for instance, passing in an async function, or an arrow function with an implicit return. 

I could be persuaded to make them void, though.